File: Heat/Models/ManifacturerModel.vb
Namespace Models

    ''' <summary>
    ''' Rappresenta un Modello di caldaia per il Produttore
    ''' </summary>
    ''' <remarks></remarks>
    Public Class ManifacturerModel
        Property ID As Integer
        Property ManifacturerID As Integer
        Property Manifacturer As manifacturer
        Property Model As String
        Property Services As List(Of BoilerService)

    End Class
End Namespace

File: Heat.ConvertedToC#/Models/ManifacturerModel.cs
using System.Collections.Generic;
namespace Heat.Models
{

    /// <summary>
    /// Rappresenta un Modello di caldaia per il Produttore
    /// </summary>
    /// <remarks></remarks>
    public class ManifacturerModel
	{
		public int ID { get; set; }
		public int ManifacturerID { get; set; }
		public Manifacturer Manifacturer { get; set; }
		public string Model { get; set; }
		public List<BoilerService> Services { get; set; }

	}
}
